docs/index.html

```diff
@@ -41,67 +41,67 @@ <h1>Platzhalter.io - eine Web App für den Umgang mit den Covid19 Restriktionen<
-        </p>
-        <p>
-          "Platzhalter.io" vereint dabei einfache Bedienbarkeit mit Datensparsamkeit, welche wiederum das Geschäfts -und Zusammenleben trotz
-          Covid19 erlaubt.
-            </p>
-        <p>
-           Wenn "Platzhalter.io" eine Erleichterung ist, dann kann der Nutzer spenden. 
+          <h3>Anwendungsfall - Gastronomie </h3> 
+        Sie haben bereits ein funktionierendes Buchungssystem aber Sie müssen Personendaten fürs Gesundheitsamt vorhalten. 
+        Sie erledigen das aktuell mit Zettel und Stift. Das ist unangenehm für Ihre Besucher und für Sie. 
+        Bieten  Sie einfach die kostenlose, datensparsame digitale Gästeliste von platzhalter.io an. Die Personendaten werden vier Wochen vorgehalten und dann automatisch gelöscht.
+       
+        <h3>Anwendungsfall - Camping Platz</h3> 
+        Sie haben bereits ein funktionierendes Buchungssystem aber Sie müssen Personendaten Personendaten fürs Gesundheitsamt vorhalten. Außerdem ist Ihr Platz aktuell sehr gefragt - Sie müssen ständig am Telefon sein. Vor Ihrem Platz herrscht das Chaos weil dieses Jahr der Campingsommer ist.
+        Tragen Sie ihre Verfügbarkeiten in die Camping Karte von platzhalter.io ein und  Ihre Besucher können sich selbst informieren, ob noch Platz ist. Außerdem werden die Personendaten Ihrer Besucher 4 Wochen vorgehalten und dann automatisch gelöscht. 
+
+        Video der Camping Karte
+        
+        <h3>Anwendungsfall - öffentliche Stell-und Parkplätze der Kommune </h3>  
+        Sie sind eine Kommune, die einen Badesee verwaltet. Dort gibt es nur begrenzt Parkplätze. Es staut sich und Chaos entsteht. Tragen Sie ihre Verfügbarkeiten in die Stellplatz Karte von platzhalter.io ein und Ihre Besucher können sich selbst informieren und sich so einen Alternativplan zurechtlegen. Die Personendaten derer, die kommen können, werden vier Wochen vorgehalten und dann automatisch gelöscht.
+        
+        <h3>Anwendungsfall - Körpernahe Dienstleistungen, zum Beispiel Frisör </h3> 
+        Durch die Covid19 Auflagen werden bei Ihnen alle Termine nur noch per Telefon gebucht. Dazu müssen alle personenbezogenen Daten abgefragt werden und bei Nachfrage dem Gesundheitsamt übermittelt werden. 
+        Das führt zu Zettelwirtschaft und Koordinations-overhead. Mit platzhalter.io hinterlegen Sie  Ihre Kapazitäten in der Web App. 
+        Ihr Besucher wiederum surft auf Ihre Website, gibt seinen Buchungswunsch ein und erhält ein QR Ticket auf sein Mobilgerät. Die Personendaten des Kunden werden - nach seinem Einverständnis - automatisch per Mobilgerät in Ihre Datenbank übertragen. 
+        Sollte ein Covid19 Fall auftreten, können Sie per Knopfdruck die Daten an das Gesundheitsamt übermitteln. Die Daten werden nach 4 Wochen automatisch gelöscht.
```

Comment:
Instead of `<p> <h3>...title...</h3> ...text... <h3>...title...</h3> ....text... </p>` it should be: `<h3>title</h3><p>text</p> <h3>title</h3> <p>text</p> ...`